portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>loas-zous</deploy.projectId>
```

Comment:
I'm not sure if this is the correct project ID. I think it should be 'zous-step-2020'.

Reply:
Right, it should be "zous-step-2020". I just set up the project here:

https://pantheon.corp.google.com/home/dashboard?project=zous-step-2020